src/dsl/verifier/proxy/stateHandler/stateHandler.ts

```diff
@@ -1,6 +1,6 @@
 import express from 'express';
 
-import { ProviderState, ProxyOptions } from '../types';
+import { ProxyOptions, ProviderState } from '../types';
```

Comment:
Is this done automatically by a plugin, and if yes, which one? I'm keen to use it!

Reply:
It might be? I have the standard eslint and prettier VS code plugins. 

I also have these relevant workspace settings:

```json
  "typescript.suggest.autoImports": true,
  "typescript.preferences.importModuleSpecifier": "relative"
```

It's also possible during refactoring I removed one of them and brought it back, and it just appended!